src/components/form/InputBase.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from "prop-types";
+import theme from "@/themes/theme";
+
+const {
+  form: { input },
+} = theme;
+
+const InputBase = ({
+  className,
+  col,
+  hasError,
+  wrapper,
+  name,
+  placeholder,
+}) => {
+  return (
+    <div className={[wrapper, col && input.col[col]]}>
+      <input
+        type="text"
+        name={name}
+        id={name}
+        css={[
+          input.base,
+          input.border,
+          input.size,
+          input.space,
+          input.focus,
+          input.error,
+          hasError && input.error,
+          className,
+        ]}
+        placeholder={placeholder}
+        ariaInvalid={!!hasError}
+        ariaDescribedby={`${name}-error`}
+      />
+    </div>
+  );
+};
+
+InputBase.propTypes = {
+  className: PropTypes.string,
+  col: PropTypes.oneOf([
+    "full",
+    "half",
+    "third",
+    "twoThirds",
+    "1",
+    "1/2",
+    "1/3",
+    "2/3",
+  ]),
```

Comment:
Please note these changes and let me know your thoughts.  My idea was to offer both options (a number or the full word) for the prop value.